framework/src/modules/auth/module.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import { BaseModule } from '..';
+import {
+	TransactionExecuteContext,
+	TransactionVerifyContext,
+	VerificationResult,
+	VerifyStatus,
+} from '../../node/state_machine';
+import { AuthAPI } from './api';
+import { RegisterMultisignitureCommand } from './commands/register_multisigniture';
+import { MODULE_ID_AUTH } from './constants';
+import { AuthEndpoint } from './endpoint';
+import { configSchema } from './schemas';
+
+export class AuthModule extends BaseModule {
+	public id = MODULE_ID_AUTH;
+	public name = 'auth';
+	public api = new AuthAPI(this.id);
+	public endpoint = new AuthEndpoint(this.id);
+	public configSchema = configSchema;
+	public commands = [new RegisterMultisignitureCommand()];
```

Comment:
probably it needs to have `moduleID` as input for each command

Reply:
`BaseCommand` has no `moduleID` param in its constructor like for example `BaseAPI` does, should I still put it there without a convention in the `BaseCommand` constructor?

Reply:
It's ok if we add it later, but to access the storage, it will require the moduleID. (we can consider adding to base command later if needed

Reply:
then let's add it now to set an example for other modules and `BaseCommand`? Adding it now without `BaseCommand` changes wouldn't hurt as you say.

Reply:
yea sounds good!

Reply:
Actually it hurts 😄 We cannot pass the `moduleID` without adding it to constructor as a parameter.

Reply:
It gives the error `Expected 0 arguments but got 1`.

Reply:
yes, we need to add the constructor if it's not same as base command

Reply:
Yes, we need to add constructor because if I don't pass the `moduleID` to the command we cannot make it a working command and can't proceed with other issues of the module epic. Adding the `moduleID` and constructor asap.